freeglut/freeglut/src/mswin/fg_main_mswin.c

```diff
@@ -1023,6 +1104,7 @@ LRESULT CALLBACK fgPlatformWindowProc( HWND hWnd, UINT uMsg, WPARAM wParam, LPAR
         break;
 #endif
 
+#if _MSC_VER > 1400
```

Comment:
Perhaps make this `#ifdef WM_SETCURSOR` as I don't believe this has changed for multiple versions, and IIRC, this built pre-Visual Studio 2005... so I don't know what changed, but under that assumption, I worry that two VS 2005 or pre-2005 builds could be different.